django/db/models/fields/related.py

```diff
@@ -583,12 +583,14 @@ def deconstruct(self):
             kwargs['parent_link'] = self.remote_field.parent_link
         # Work out string form of "to"
         if isinstance(self.remote_field.model, str):
-            kwargs['to'] = self.remote_field.model
+            # Force model name to lower case
+            if "." in self.remote_field.model:
+                app_label, _dot, rest = self.remote_field.model.partition(".")
+                kwargs['to'] = "%s.%s" % (app_label, rest.lower())
+            else:
+                kwargs['to'] = self.remote_field.model.lower()
```

Comment:
I think we can simplify this and use:
```python
kwargs['to'] = self.remote_field.model.lower()
```
like we already do in [`Operation`](https://github.com/django/django/blob/895f28f9cbed817c00ab68770433170d83132d90/django/db/migrations/operations/base.py#L128-L134).